### ISSUE TITLE
missing property 'removed_by_category' on Submission type

### ISSUE THREAD

**Issue by prazdevs** [not-an-aardvark/snoowrap#273]
If i run through pure Javascript, i can access the property named `removed_by_category` which gives me info of a removal reason of a submission (deleted by owner or removed by a moderator).

But it doesnt appear on the type of submission and i cannot access it while using typescript :/

**Comment by olafurw** [comment]
Same for `getContentByIds`

Should be something like this.
```
getContentByIds(ids: string[]): Promise<_Listing<_Submission | _Comment>>;
```

**Comment by prazdevs** [comment]
I'll look into it given i have the time, and might look into both of those missing properties, if you have any other, let me know!

**Comment by SpyTec** [comment]
@olafurw you can open a new issue for that, closing this one as `removed_by_category` is implemented in #281
